variables.tf

```diff
@@ -160,5 +160,10 @@ variable "simpheraInstances" {
   }))
 
   description = "A list containing the individual SIMPHERA instances, such as 'staging' and 'production'."
-  default     = {}
+}
+
+variable "apiServerAuthorizedIpRanges" {
+  type        = set(string)
+  description = "List of authorized IP address ranges that are granted access to the Kubernetes API server."
```

Comment:
Maybe add an example on how to define  these ranges.

Reply:
Done